packages/main/src/ResponsivePopover.js

```diff
@@ -116,11 +116,12 @@ class ResponsivePopover extends Popover {
 	/**
 	 * Shows popover on desktop and dialog on mobile.
 	 * @param {HTMLElement} opener the element that the popover is shown at
+	 * @param {boolean} preventInitialFocus Prevents applying the focus inside the popup
 	 * @public
 	 * @async
 	 * @returns {Promise} Resolves when the responsive popover is open
 	 */
-	async showAt(opener) {
+	async showAt(opener, preventInitialFocus) {
```

Comment:
```suggestion
	async showAt(opener, preventInitialFocus = false) {
```
Can you default the `preventInitialFocus` parameter to false? This will look nicer in our Web Components for React docs as it will be marked as optional then